agent/consul/state/state_store.go

```diff
@@ -203,6 +203,28 @@ func (s *Store) Snapshot() *Snapshot {
 	return &Snapshot{s, tx, idx}
 }
 
+// WalkAllTables basically lets you dump memdb generically and exists primarily
+// for very specific types of unit tests and should not be executed in
+// production code.
+func (s *Store) WalkAllTables(fn func(table string, item interface{}) bool) error {
```

Comment:
If this is used for tests, maybe this could be in a test file instead of a production file?

Reply:
It's used for tests in other packages so that wouldn't work.